content/en/docs/instrumentation/java/manual.md

```diff
@@ -157,38 +557,158 @@ instrumenting][instrumentation library] the [instrumented library] or
 application to be monitored. More information is available in the specification
 chapter [Obtaining a Tracer].
 
+Anywhere in your application where you write manual tracing code should call
+`getTracer` to acquire a tracer. For example:
+
 ```java
-import io.opentelemetry.api;
+import io.opentelemetry.api.trace.Tracer;
 
-//...
+Tracer tracer = openTelemetry.getTracer("instrumentation-scope-name", "instrumentation-scope-version");
+```
+
+The values of `instrumentation-scope-name` and `instrumentation-scope-version`
+should uniquely identify the
+[instrumentation scope](/docs/specs/otel/glossary/#instrumentation-scope), such
+as the package, module or class name. While the name is required, the version is
+still recommended despite being optional. Note, that all `Tracer`s that are
+created by a single `OpenTelemetry` instance will interoperate, regardless of
+name.
+
+It's generally recommended to call `getTracer` in your app when you need it
+rather than exporting the `tracer` instance to the rest of your app. This helps
+avoid trickier application load issues when other required dependencies are
+involved.
 
-Tracer tracer =
-    openTelemetry.getTracer("instrumentation-library-name", "1.0.0");
+In the case of the [example app](#example-app), there are two places where a
+tracer may be acquired with an appropriate instrumentation scope:
+
+First, in the `index` method of the `RollController` as follows:
+
+```java
+package otel;
+
+// ...
+import org.springframework.beans.factory.annotation.Autowired;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.Tracer;
+
+@RestController
+public class RollController {
+  private static final Logger logger = LoggerFactory.getLogger(RollController.class);
+  private final Tracer tracer;
+
+  @Autowired
+  RollController(OpenTelemetry openTelemetry) {
+    tracer = openTelemetry.getTracer(RollController.class.getName(), "0.1.0");
+  }
+  // ...
+}
 ```
 
-Important: the "name" and optional version of the tracer are purely
-informational. All `Tracer`s that are created by a single `OpenTelemetry`
-instance will interoperate, regardless of name.
+And second, in the _library file_ `Dice.java`:
+
+```java
+// ...
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.Tracer;
+
+public class Dice {
+
+  private int min;
+  private int max;
+  private Tracer tracer;
+
+  public Dice(int min, int max, OpenTelemetry openTelemetry) {
+    this.min = min;
+    this.max = max;
+    this.tracer = openTelemetry.getTracer(Dice.class.getName(), "0.1.0");
+  }
+
+  public Dice(int min, int max) {
+    this.min = min;
+    this.max = max;
+    this.tracer = OpenTelemetry.noop().getTracer(Dice.class.getName(), "0.1.0");
```

Comment:
Replace with: `this(min, max, OpenTelemetry.noop())`

Reply:
ah, my Java is rusty ;-)

I am wondering, would it make sense here to replace `OpenTelemetry.noop()` with `GlobalOpenTelemetry`. If I understand things correctly, this would add another layer of fallback, because if a global object is registered it can be used, otherwise it is a noop as well.

Reply:
Potentially... It definitely provides some small increased chance that a valid OpenTelemetry instance is used. Not sure how to weigh that against the fact that we advise against using GlobalOpenTelemetry.

Reply:
Thanks, need to think about that one. On the one hand I like additional layers of fallback, but since these are the official docs it is very likely that people copy from them into their libraries and then a pattern emerges that GlobalOpenTelemetry is used because it is easy, so if we want people to _not_ use it, we should propagate the noop one

Reply:
fixed, I went with the noop. There is a note about GlobalOpenTelemetry below, shall we keep that one?

Reply:
You can keep it, but may want to use some the language we use in the [GlobalOpenTelemetry javadoc](https://github.com/open-telemetry/opentelemetry-java/blob/ebf96dc5de85676a4d2d29e8ea0be7f49b3ed6b0/api/all/src/main/java/io/opentelemetry/api/GlobalOpenTelemetry.java#L34-L37) so that we're consistently warning against using it.

Reply:
@jack-berg , @mateuszrzeszutek you are the experts here, so I'll remove/add/change it to whatever works for you.

Note that the manual instrumentation guide is right now not talking a lot about combining the agent with manual instrumentation. We might need to do another round on thinking about how to incorporate that best (maybe it requires it's own section somewhere in the "Setup" section?)

Reply:
> We might need to do another round on thinking about how to incorporate that best (maybe it requires it's own section somewhere in the "Setup" section?)

I think even a small section that focuses just on that would be a good idea 👍 
Definitely can be a separate PR though

Reply:
Follow-up issue #3270